### PR TITLE
docs: runtime config store

### PR DESCRIPTION
**Pull request by jakmeier**
small and steady steps to replace all the TODO's I left in the docs :)

### CODE REVIEW

File: docs/architecture/gas/parameter_definition.md [hunk 1]
```diff
@@ -10,7 +10,27 @@ starting from version 53. To see all parameter values for a specific version at
 once, check out the list of JSON snapshots generated in this directory:
 [core/primitives/src/runtime/snapshots](https://github.com/near/nearcore/blob/d0dc37bf81f7e7bde9c560403b085fae04108659/core/primitives/src/runtime/snapshots).
 
-<!-- TODO: Runtime Config Store -->
+## Using Parameters in Code
+
+The introduction on this page already hints at it. Parameter values are
+versioned. In other words, they can change between protocol versions. A nearcore
```
Comment (akhi3030) [inline docs/architecture/gas/parameter_definition.md:16]:
```suggestion
As the introduction on this page already hints at it, parameter values are
versioned. In other words, they can change between protocol versions. A nearcore
```

Comment (akhi3030) [inline docs/architecture/gas/parameter_definition.md:16]: I believe this to be the case and I wonder if we should specify that any changes to parameter values must happen via protocol upgrades.

File: docs/architecture/gas/parameter_definition.md [hunk 2]
```diff
@@ -10,7 +10,27 @@ starting from version 53. To see all parameter values for a specific version at
...
+binary has to support multiple versions and choose the correct parameter value at
+runtime.
+
+To make this easy, there is
+[`RuntimeConfigStore`](https://github.com/near/nearcore/blob/a8964d200b3938a63d389263bc39c1bcd75b1de4/core/primitives/src/runtime/config_store.rs#L43).
+It contains a sparse map from protocol versions to complete runtime
+configurations (`BTreeMap<ProtocolVersion, Arc<RuntimeConfig>>`).
+The runtime then uses `store.get_config(protocol_version)` to access a runtime
```
Comment (akhi3030) [inline docs/architecture/gas/parameter_definition.md:24]:
```suggestion
The runtime then uses `store.get_config(protocol_version)` to access the runtime
```

**Review by akhi3030 — APPROVED** [review]

File: docs/architecture/gas/parameter_definition.md (hunk 2 above)
Comment (jakmeier) [reply docs/architecture/gas/parameter_definition.md:24]: Hm, it's one runtime config of many in the list, so I think `a` is also correct here and it puts more emphasis on the fact that there are multiple configs to choose from.

File: docs/architecture/gas/parameter_definition.md (hunk 1 above)
Comment (jakmeier) [reply docs/architecture/gas/parameter_definition.md]: Yeah you are right. I though I already described it like you say. Now I rephrased slightly to make it clearer that parameter changes can only happen if we also change the protocol version.

File: docs/architecture/gas/parameter_definition.md (hunk 2 above)
Comment (akhi3030) [reply docs/architecture/gas/parameter_definition.md:24]: Ah you are right then.

**Pull request merged**